packages/styles/src/types.ts

```diff
@@ -411,6 +411,9 @@ export type HvThemeComponents = {
     thumbnailBorderRadius: CSSProperties["borderRadius"];
     thumbnailSelectedBorder: CSSProperties["border"];
   };
+  drawer: {
+    backDropBackgroundColor: CSSProperties["backgroundColor"];
```

Comment:
No real reason other then it just felt that it should be something "themable" and thus easily accessible on theme editor, i can remove it if no one else agrees

Reply:
I guess this is somewhat subjective, so I'm not gonna oppose it 👍 

To me, it does make lots of sense to have a backdrop color customisation - but for all backdrops, not `HvDrawer` specifically.

We already have a `theme.colors.shadow`. We probably should have a `theme.colors.backdrop` and reuse it across components that need one (eg. `HvDialog`).
This would also eliminate the issue issue of the alpha channel with css variables, as we could simply (as we're doing with `.shadow`) use `rgba` directly.

Reply:
I agree with the general backdrop color for a theme, but the issue of the alpha channel i think would still remain when changing themes, unless we hardcode a value for light and dark and we would use the ability to add the alpha in reference to a color in the theme. In shadows this is okay because they are usually the same color, I don't know if the same applies here

Reply:
Being it part of the "base" `theme.colors` palette, then I argue they _should_ be hardcoded, and not dependent on other colors - similara to `shadow` and other colors we have (ie. `primary_20` is `primary_80` with a different alpha in both themes).

But we can look into this later. I'm ok with this. 👍